group_vars/nibbler_cluster/vars.yml

```diff
@@ -4,6 +4,8 @@ slurm_cluster_domain: ''  # Only add hpc.rug.nl domain when jumphost is register
 stack_prefix: 'nb'
 slurm_version: '20.11.8-1.el7.umcg'
 repo_manager: 'pulp'
+server_dns: 'umcg-icat01.hpc.rug.nl'
```

Comment:
These server_dns and server_dns_short variables are a bit confusing as their location here suggests this applies to all Nibbler machines. There are `fqdn` (Fully Qualified Domain Name) attributes for each machine in `group_vars/nibbler_cluster/ip_addresses.yml`. The short name can be easily deduced from the FQDN using a jinja filter.